src/styles/blocks/header.scss

```diff
@@ -0,0 +1,79 @@
+.header {
+  display: flex;
+  justify-content: space-between;

```

Comment:
Will be better to use variable for correct indents between the cards (it should fix the tests)